site/pages.json

```diff
@@ -401,18 +399,7 @@
           "title": "mobi-stabi-training",
```

Comment:
This is the browser title - doesn't look good IMO.
<img width="300" alt="Bildschirmfoto 2019-11-27 um 12 34 22" src="https://user-images.githubusercontent.com/11008743/69720142-3f75f500-1112-11ea-986b-4c3933e73204.png">

Reply:
changed to "Triathlon"